app/src/main/java/com/example/myapplication2/SignInActivity.kt

```diff
@@ -24,6 +28,13 @@ class SignInActivity : AppCompatActivity() {
             insets
         }
 
+        // Add FragmentA initially
```

Comment:
Code for fragments should be located in a seperate sample activity, not in SignInActivity.